src/inference/tests/functional/caching_test.cpp

```diff
@@ -2035,7 +2037,7 @@ TEST_P(CachingTest, LoadAUTO_OneDevice) {
     });
     std::string cacheDir = m_cacheDir;
     MkDirGuard guard(cacheDir);
-    for (index; index < TEST_COUNT; index++) {
+    for (; index < TEST_COUNT; index++) {
```

Comment:
I think we can remove local variable and move it to the cycle.

Reply:
fixed